BeerOverflowWindowsApp/BarComparers/ComparerByRating.cs

```diff
@@ -0,0 +1,44 @@
+﻿using BeerOverflowWindowsApp.DataModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BeerOverflowWindowsApp.BarComparers
+{
+    class ComparerByRating : IComparer<BarData>
+    {
+        public int Compare(BarData barData1, BarData barData2)
+        {
+            int result;
+            if (barData1.Ratings == null && barData2.Ratings == null)
+            {
+                result = 0;
+            }
+            else
+            if (barData1.Ratings == null && barData2.Ratings != null)
+            {
+                result = 1;
+            }
+            else
```

Comment:
Your "else if" here is written in two lines when in 
"BeerOverflowWindowsApp/BarComparers/ComparerByDistance.cs" (code above)
it's in one line. One of these needs correction, i guess.